### PR TITLE
Add parameter ROUNDCUBEMAIL_INSTALL_PLUGINS

### PR DESCRIPTION
**Pull request by mikbund**
Added ROUNDCUBEMAIL_INSTALL_PLUGINS

This way plugins will be downloaded during initialization and be ready to be enabled.

From my docker-compose.yml:
 - ROUNDCUBEMAIL_INSTALL_PLUGINS=roundcube/carddav,kitist/html5_notifier,texxasrulez/vcard_attach

This downloads the plugins.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
Hi @thomascube 

This seems reasonable, what do you think ?

Because having people do custom Dockerfiles makes people have not up to date docker images :/

**Comment by williamdes** [comment]
Hi @thomascube
I can work on this one this week, let me know if I can help or something need to be done

**Comment by thomascube** [comment]
@williamdes Thanks for looking into this. First I wasn't sure if this is the right approach for managing plugins but with that feature one does not have to either build custom images or use a persistent storage volume. So please go on.

File: apache/docker-entrypoint.sh [hunk 1]
```diff
@@ -147,4 +163,9 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 
 fi
 
+
+
+
+
+
```
Comment (thomascube) [inline apache/docker-entrypoint.sh:170]: Remove this useless white-space

File: apache/docker-entrypoint.sh [hunk 2]
```diff
@@ -71,6 +71,22 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
   : "${ROUNDCUBEMAIL_SKIN:=elastic}"
   : "${ROUNDCUBEMAIL_TEMP_DIR:=/tmp/roundcube-temp}"
 
+
+
+    # add plugins
+    echo "adding plugins from list"
+    echo ${ROUNDCUBEMAIL_INSTALL_PLUGINS}
```
Comment (thomascube) [inline apache/docker-entrypoint.sh:78]: These echo lines should go into the if clause.

**Comment by williamdes** [comment]
> @williamdes Thanks for looking into this. First I wasn't sure if this is the right approach for managing plugins but with that feature one does not have to either build custom images or use a persistent storage volume. So please go on.

Task done: https://github.com/roundcube/roundcubemail-docker/pull/245 🎉

**Comment by williamdes** [comment]
Please test https://github.com/roundcube/roundcubemail-docker/pull/250

**Pull request closed without merging**